src/styles/MyInput.js

```diff
@@ -0,0 +1,11 @@
+import { TextField } from "@mui/material";
```

Comment:
```suggestion
import TextField from "@mui/material/TextField";
```

https://mui.com/material-ui/guides/minimizing-bundle-size/#development-environment